### PR TITLE
feat(rebrand): rebrand recommended-workflow to standard-version

### PR DESCRIPTION
**Pull request by bcoe**
BREAKING CHANGE: changes name of module from conventional-recommended-workflow to standard-version

### CODE REVIEW

**Comment by nexdrew** [comment]
GitHub URLs in package.json need to be fixed. :)

**Comment by stevemao** [comment]
Love the new name :)

**Comment by bcoe** [comment]
@stevemao, yes! @nexdrew and I had a discussion, we were thinking we'd keep the scope of the `standard-changelog` bin to just changelog generation. This bin can replace the `npm version` workflow, with changelog generation as part of it.

File: package.json [hunk 1]
```diff
@@ -1,27 +1,31 @@
 {
-  "name": "conventional-recommended-workflow",
+  "name": "standard-version",
   "version": "1.0.0",
-  "description": "apply the recommended workflow outlined in conventional-changelog-cli.",
+  "description": "replacement for `npm version` with automatic CHANGELOG generation",
   "bin": "index.js",
   "main": "index.js",
```
Comment (stevemao) [inline package.json]: We can remove `main` here?

**Pull request merged**